### PR TITLE
Usagov 1958 document analytics

### PR DESCRIPTION
**Pull request by omerida**
## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-19958

## Description

Update the links in the life events grid to have the same data attribute for analytics that the search results do.
Documents the data attributes as CSS selectors to be handled with care.

## Type of Changes

- [ ] New Feature
- [x] Bugfix
- [ ] Frontend (Twig, Sass, JS)
  - Add screenshot showing what it should look like
- [x] Drupal Config (requires "drush cim")
- [ ] New Modules (requires rebuild)
- [ ] Documentation
- [ ] Infrastructure
  - [ ] CMS
  - [ ] WAF
  - [ ] Egress
  - [ ] Tools
- [ ] Other

### Change Requirements

- [ ] Requires New Documentation (Link: {})
- [x] Requires New Config Updated view
- [ ] Requires New Content

### Validation Steps

- Load the benefit search page, inspect the titles for life events in the grid above the search results. The `<a>` links should have a `data-analytics="life-events-links"` attribute

## Security Review

- [ ] Adds/updates software (including a library or Drupal module)
- [ ] Communication with external service
- [ ] Changes permissions or workflow
- [ ] Requires SSPP updates

## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.

### CODE REVIEW

File: config/sync/views.view.life_events_grid.yml [hunk 1]
```diff
@@ -229,7 +282,7 @@ display:
           exclude: false
           alter:
             alter_text: false
-            text: ''
+            text: '<a href="{{ field_b_search_title__value }}" data-analytics=">{{ field_b_search_title }}</a>'
```
Comment (omerida) [inline config/sync/views.view.life_events_grid.yml]:
```suggestion
            text: ''
```

**Comment by YaritzaGarcia** [comment]
@omerida Hello! When I was reviewing I noticed that all the life events in the grid have 2 <a> but the first one (Death) just have one. Is this intentional? This doesn't happen in the spanish site.
 
Screenshot 2024-09-24 at 2 22 21 PM
Screenshot 2024-09-24 at 2 22 35 PM

**Comment by omerida** [comment]
i'll take a look. This looks like an old bug I thought I squashed before.

**Comment by omerida** [comment]
@YaritzaGarcia thanks for catching that bug. I've updated the view to prevent the nested link.

**Review by YaritzaGarcia — APPROVED** [review]

**Pull request merged**